modular_skyrat/modules/customization/game/objects/items/plushes.dm

```diff
@@ -294,3 +294,13 @@
 	attack_verb_continuous = list("sneezes on", "detains", "tazes")
 	attack_verb_simple = list("sneeze on", "detain", "taze")
 	squeak_override = list('modular_skyrat/modules/emotes/sound/emotes/female/female_sneeze.ogg' = 1)
+
+/obj/item/toy/plush/xixi
+	name = "familiar looking harpy plushie"
```

Comment:
should just be "familiar harpy plushie" honestly.

Reply:
name was request of the person who posted the bounty, i'll ask them about it, but otherwise- no need to change rn